docs/source/conf.py

```diff
@@ -0,0 +1,55 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'fairscale'
+copyright = '2020, Facebook AI Research'
+author = 'Facebook AI Research'
+
+# The full version, including alpha/beta/rc tags
+release = '0.0.2'
```

Comment:
can this be imported so that all version numbers are in the same place or at least cross-ref them on all places so we don't forget to change all of them together?

Reply:
Yes. Will add that in a future change. Learning this as I'm doing it.